spec/poros/quote_spec.rb

```diff
@@ -0,0 +1,28 @@
+
+
+require "rails_helper"
+
+RSpec.describe Quote do
+  it "exists" do
+    data =  {
+                "_id": "r3wAKE9N-Nei",
+                "content": "Imagination will often carry us to worlds that never were. But without it we go nowhere.",
+                "author": "Carl Sagan",
+                "tags": [
+                    "famous-quotes"
+                ],
+                "authorId": "W2oRVCp-mXLC",
+                "authorSlug": "carl-sagan",
+                "length": 88,
+                "dateAdded": "2020-02-07",
+                "dateModified": "2020-02-07"
+            }
+
+    quote = Quote.new(data)
```

Comment:
This is a great way to test your poro. Sometimes I'll see folks make a call to the service or facade here, but creating your own data set to test with is the way to go. You have so much more control over your testing this way.